docs/workloads/azure/frontend/intro_node_js.md

```diff
@@ -4,6 +4,12 @@ title: Introduction
 sidebar_label: Introduction
 hide_title: false
 hide_table_of_contents: true
+description: Introduction
+keywords:
+  - rest 
+  - api
+  - csr
+  - ssr
```

Comment:
It would be good to have the full description of the csr and ssr

Reply:
description added